contracts/InterestRate/StableRate.sol

```diff
@@ -0,0 +1,248 @@
+    bool public constant isInterestRateModel = true;
+
+    uint256 private constant BASE = 1e18;
+
+    /**
+     * @notice The approximate number of blocks per year that is assumed by the interest rate model
+     */
+    uint256 public constant blocksPerYear = 2102400;
+
+    /**
+     * @notice The stable base interest rate which is the y-intercept when utilization rate is 0
+     */
+    uint256 public baseRatePerBlockForStable;
+
+    /**
+     * @notice The multiplier of utilization rate that gives the slope of the interest rate
+     */
+    uint256 public multiplierPerBlock;
+
+    /**
+     * @notice The multiplier of utilization rate that gives the slope of the stable interest rate
+     */
+    uint256 public multiplierPerBlockForStable;
+
+    /**
+     * @notice The multiplierPerBlock after hitting a specified utilization point for stable rate
+     */
+    uint256 public jumpMultiplierPerBlockForStable;
+
+    /**
+     * @notice The utilization point at which the jump multiplier is applied for stable rate
+     */
+    uint256 public kink;
+
+    /**
+     * @notice The premium rate applicable on optimal stable loan rate
+     */
+    uint256 public stableRatePremium;
+
+    /**
+     * @notice Optimal Stable Loan Rate
+     */
+    uint256 public optimalStableLoanRate;
+
+    /**
+     * @notice The address of the owner, i.e. the Timelock contract, which can update parameters directly
+     */
+    address public owner;
+
+    /**
+     * @param _baseRatePerBlockForStable The approximate target base APR, as a mantissa (scaled by BASE)
+     * @param _multiplierPerBlock The rate of increase in interest rate wrt utilization (scaled by BASE)
+     * @param _multiplierPerBlockForStable The rate of increase in interest rate wrt utilization (scaled by BASE)
+     * @param _jumpMultiplierPerBlockForStable The multiplierPerBlock after hitting a specified utilization point
+     * @param _kink The utilization point at which the jump multiplier is applied
+     * @param _stableRatePremium The multiplierPerBlock after hitting a specified utilization point
+     * @param _optimalStableLoanRate Optimal stable loan rate percentage.
+     */
+    constructor(
+        uint256 _baseRatePerBlockForStable,
+        uint256 _multiplierPerBlock,
+        uint256 _multiplierPerBlockForStable,
+        uint256 _jumpMultiplierPerBlockForStable,
+        uint256 _kink,
+        uint256 _stableRatePremium,
+        uint256 _optimalStableLoanRate,
+        address owner_
+    ) {
+        owner = owner_;
+
+        updateStableRateModelInternal(
+            _baseRatePerBlockForStable,
+            _multiplierPerBlock,
+            _multiplierPerBlockForStable,
+            _jumpMultiplierPerBlockForStable,
+            _kink,
+            _stableRatePremium,
+            _optimalStableLoanRate
+        );
+    }
+
+    /**
+     * @notice Update the parameters of the interest rate model (only callable by owner, i.e. Timelock)
```

Comment:
use `custom:access` for permissions

Reply:
Fixed, can check the changes at [this commit](2981e0d08b247b7c7a7dc3461e9701d7d593c520).